core/src/core.cpp

```diff
@@ -337,7 +337,7 @@ int sdrpp_main(int argc, char* argv[]) {
 #else
     const char* glsl_version = "#version 120";
     GLFWmonitor* monitor = NULL;
-    for (int i = 0; i < OPENGL_VERSION_COUNT; i++) {
+    for (size_t i = 0; i < OPENGL_VERSION_COUNT; i++) {
```

Comment:
please revert, that value  (manually entered, it's a) would never get passed 5 or 6 there's no reason to use size_t which just takes more space for no reason